src/map/packets/change_music.cpp

```diff
@@ -35,6 +35,6 @@ CChangeMusicPacket::CChangeMusicPacket(uint8 BlockID, uint8 MusicTrackID)
     this->setType(0x5F);
     this->setSize(0x08);
 
-    ref<uint8>(0x04) = BlockID;      // block
-    ref<uint8>(0x06) = MusicTrackID; // music
+    ref<uint16>(0x04)  = BlockID;     // block
```

Comment:
CI fails because of styling, like this one. Please correct those alignments

Reply:
![imagen](https://github.com/LandSandBoat/server/assets/60053999/4c709ef3-360b-4c62-8268-4e39793dcf84)

Reply:
Fixed